astro/src/content/blog/digital-identity.mdx

```diff
@@ -0,0 +1,55 @@
+---
+publish_date: 2024-02-20
+title: Digital Identity - Human Elements of a Connected World
+description: Your identity is so much more than the pieces of paper and plastic that sit in a wallet. In today's world, your digital identity is valuable, and ever-changing.
+authors: Brad McCarty
+image: /img/blogs/digital-identity/digital-identity.png
+categories: Education
+tags: digital identity
+excerpt_separator: "{/* more */}"
+---
+Every time you key in credentials to access your go-to app, sift through your inbox, scroll social media, or jump into an [online game](industries/gaming-entertainment), you're doing more than just inputting a username and password.{/* more */} You're affirming your digital identity, declaring, "Yep, that's me!" And here's the kicker: we're all engaging in this ritual almost daily, likely thousands of times each month. But we seldom pause to ponder the dance happening behind the screen. So let's unpack the layers around digital ID, the daily login routine, CIAM providers, and the world of authentication.
+
+## What is Digital Identity?
+
+Imagine identity as the digital equal to your DNA. It contains every choice, preference, and action you make online. It's your virtual version of all of your personal data, evolving with every click, every preference tweak, and every new interaction. This identity isn't static. It's a living, breathing entity in the digital world, reshaping itself as you navigate through your online life.
+
+Now that we've painted that picture, let's talk about the different elements that help you to complete those actions. From the first login to personalization, each step plays a critical role in your experience.
+
+## User Login: Saying Hello
+
+You can picture logging in as the initial fist bump or high five you exchange with a platform. This is your chance to present your digital "Hi, my name is..." sticker.
+
+But the real magic happens during the [authentication](/articles/authentication/) process. This is where the platform takes a moment to ensure you're really you. They might use something as simple as a password, a code pinged to your phone, or even your unique [fingerprint](/articles/authentication/passwordless-authentication-security). The goal is to establish trust that it's actually you on the other end of the process.
+
+## Authentication: You Shall (Not?) Pass
+
+Think of authentication as the gatekeeper. It's standing guard, ensuring that every claim of identity is legit. This process is a critical checkpoint that balances ease of access with the need for security. Today's Internet is full of [digital threats](/blog/grammarly-proves-ciam-not-optional). So the art and science of authentication are constantly advancing. We're always adopting new forms, like biometrics, [multi-factor authentication](/articles/authentication/multi-factor-authentication), and passwordless technologies. It's a persistent back and forth, where the good guys are always trying to outsmart the bad actors.
```

Comment:
This is very close to the line on Authentication vs Authorization. Gatekeeping to me falls more along the lines of Authorization.  You could change the header to Authentication & Authorization.  Then it becomes who you are and what you can do. That falls in line with gatekeeping. You are Brad and you can pass.

Reply:
Done. Thanks.